deploy/charts/cert-manager/values.yaml

```
@@ -232,6 +232,17 @@ webhook:
   # rules or requiring NET_BIND_SERVICE capabilities to bind port numbers <1000
   securePort: 10250
 
+  # Specifies if the webhook should be started in hostNetwork mode.
+  #
+  # Required for use in managed kubernetes clusters (such as AWS EKS) with custom
```

```suggestion
  # Required for use in some managed kubernetes clusters (such as AWS EKS) with custom
```

Not want to rule our k8s solutions who allow custom networking on controller nodes :)